.github/workflows/validate_pr.yml

```diff
@@ -0,0 +1,19 @@
+name: Validate PR
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-test-package:
+    name: Build and Package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Build and Package with Maven
+        run: ./mvnw clean -U install
```

Comment:
Can we add `--no-transfer-progress` here as well ?

Reply:
@mohan-13,
Added the no-transfer-progress argument, please check now.